DuckDuckGo/TabViewController.swift

```diff
@@ -317,96 +288,24 @@ class TabViewController: UIViewController {
 
         return activities
     }
-    
+
     func initAttributionLogic() {
         adClickAttributionLogic.delegate = self
         adClickAttributionDetection.delegate = adClickAttributionLogic
     }
-    
-    // swiftlint:disable function_body_length
-    func initUserScripts() {
-        
-        let currentMainRules = ContentBlocking.contentBlockingManager.currentMainRules
-        let privacyConfig = ContentBlocking.privacyConfigurationManager.privacyConfig
-        
-        let contentBlockerConfig = DefaultContentBlockerUserScriptConfig(privacyConfiguration: privacyConfig,
-                                                                         trackerData: currentMainRules?.trackerData,
-                                                                         ctlTrackerData: nil,
-                                                                         tld: Self.tld,
-                                                                         trackerDataManager: ContentBlocking.trackerDataManager)
-        let contentBlockerRulesScript = ContentBlockerRulesUserScript(configuration: contentBlockerConfig)
-        contentBlockerUserScript = contentBlockerRulesScript
-        
-        let surrogates = FileStore().loadAsString(forConfiguration: .surrogates) ?? ""
-        let surrogatesConfig = DefaultSurrogatesUserScriptConfig(privacyConfig: privacyConfig,
-                                                                 surrogates: surrogates,
-                                                                 trackerData: currentMainRules?.trackerData,
-                                                                 encodedSurrogateTrackerData: currentMainRules?.encodedTrackerData,
-                                                                 trackerDataManager: ContentBlocking.trackerDataManager,
-                                                                 tld: Self.tld,
-                                                                 isDebugBuild: isDebugBuild)
-        let surrogatesScript = SurrogatesUserScript(configuration: surrogatesConfig)
-        autofillUserScript = createAutofillUserScript()
-
-        userScripts = [
-            debugScript,
-            textSizeUserScript,
-            findInPageScript,
-            navigatorPatchScript,
-            surrogatesScript,
-            contentBlockerRulesScript,
-            fingerprintScript,
-            faviconScript,
-            fullScreenVideoScript,
-            autofillUserScript,
-            printingUserScript
```

Comment:
why was documentScript (DocumentUserScript) missing here? was it intentional, should I remove it from UserScripts.swift?

Reply:
Looking back on the blame, I think this is iOS 12 leftover and could be removed, probably? :)

Reply:
ok, made extra clean-up of document.js and LongPressGestureRecognizer